frontend/src/Editor/Components/Table/Table.jsx

```diff
@@ -600,51 +601,57 @@ export function Table({
                   </span>
                 </OverlayTrigger>
               )}
-              <OverlayTrigger
-                trigger="click"
-                rootClose={true}
-                overlay={
-                  <Popover>
-                    <div
-                      data-cy={`dropdown-hide-column`}
-                      className={`dropdown-table-column-hide-common ${
-                        darkMode ? 'dropdown-table-column-hide-dark-themed' : 'dropdown-table-column-hide'
-                      } `}
-                    >
-                      <div className="dropdown-item">
-                        <IndeterminateCheckbox {...getToggleHideAllColumnsProps()} />
-                        <span className="hide-column-name" data-cy={`options-select-all-coloumn`}>
-                          Select All
-                        </span>
-                      </div>
-                      {allColumns.map((column) => (
-                        <div key={column.id}>
-                          <div>
-                            <label className="dropdown-item">
-                              <input
-                                type="checkbox"
-                                data-cy={`checkbox-coloumn-${String(column.Header).toLowerCase().replace(/\s+/g, '-')}`}
-                                {...column.getToggleHiddenProps()}
-                              />
-                              <span
-                                className="hide-column-name"
-                                data-cy={`options-coloumn-${String(column.Header).toLowerCase().replace(/\s+/g, '-')}`}
-                              >
-                                {` ${column.Header}`}
-                              </span>
-                            </label>
-                          </div>
+              {toggleShowColumns && (
```

Comment:
```suggestion
              {!hideColumnSelectorButton && (
```
This negation is required given the new property naming.